internal/cmd/testdata/scripts/chattr.txtar

```diff
@@ -6,6 +6,16 @@ exec chezmoi chattr empty $HOME${/}.file
 ! exists $CHEZMOISOURCEDIR/dot_file
 exists $CHEZMOISOURCEDIR/empty_dot_file
 
+# test that chezmoi chattr remove sets the remove attribute on a file
+exec chezmoi chattr remove $HOME${/}.file
+! exists $CHEZMOISOURCEDIR/empty_dot_file
+exists $CHEZMOISOURCEDIR/remove_dot_file
+
+# test that chezmoi chattr noremove sets the remove attribute on a file
```

Comment:
nit: s/sets the remove attribute/removes the remove attribute/

Reply:
I’ll fix this this evening.